.github/workflows/main.yml

```diff
@@ -0,0 +1,11 @@
+on: [push]
+jobs:
+  profile:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Profiler
+        uses: ./ # Uses an action in the root directory
```

Comment:
This seems like hardcoded to me, ideally we would want the directory to be chosen by the user itself right?